<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>card3</class>
 <widget class="QWidget" name="card3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#card3{
border-image: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="main" native="true">
     <property name="styleSheet">
      <string notr="true">#main{
border-image: url(:/images/sideBar/black0.png) 0 0 0 0 stretch stretch;
border-radius: 12px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="bar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="main_label">
           <property name="styleSheet">
            <string notr="true">color:rgb(55,55,55);</string>
           </property>
           <property name="text">
            <string>站点客流量预测</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1034</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="content" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="hsp3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="left" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>398</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>398</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>100</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>200</number>
            </property>
            <item>
             <widget class="QWidget" name="leftcard" native="true">
              <property name="styleSheet">
               <string notr="true">#leftcard {
        background-image: url(:/images/sideBar/black2.png);
		border-radius: 12px;
    }
</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QWidget" name="hang1" native="true">
                 <property name="styleSheet">
                  <string notr="true">QComboBox {
    border: none;
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
    color: rgb(55, 55, 55);
    padding: 5px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    border: none;
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
}

QComboBox:hover {
    border-image: url(:/images/sideBar/black3.png) 0 0 0 0 stretch stretch;
}

QComboBox QAbstractItemView {
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
    background: transparent;
}

QComboBox QAbstractItemView::item {
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
    color: rgb(55, 55, 55);
}

QComboBox QAbstractItemView::item:selected {
    border-image: url(:/images/sideBar/black3.png) 0 0 0 0 stretch stretch;
    color: rgb(55, 55, 55);
}
</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QComboBox" name="station">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>85</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>85</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QComboBox::down-arrow {
    image: url(:/images/page3/station.png);
    width: 17px;
    height: 17px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="hour">
                    <property name="minimumSize">
                     <size>
                      <width>56</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>56</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QComboBox::down-arrow {
    image: url(:/images/page0/hour.png);
    width: 17px;
    height: 17px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="hang2" native="true">
                 <property name="styleSheet">
                  <string notr="true">QDateEdit {
    border: none;
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
    color: rgb(55, 55, 55);
    padding: 5px;
}

QLabel {
	color: rgb(55,55,55);

}
</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QDateEdit" name="chooseTime">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="arrow">
                    <property name="minimumSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border-image: url(:/images/page3/arrow.png) 0 0 0 0 stretch stretch;</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>10</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>10</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="aimTime">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="hang3" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QPushButton" name="confirm">
                    <property name="minimumSize">
                     <size>
                      <width>80</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">#confirm {
        border: none;
		outline: none;
        background-image: url(:/images/sideBar/black3.png);
		border-radius: 12px;
		color: rgb(55, 55, 55);
    }
#confirm:hover {
        background-image: url(:/images/sideBar/black4.png);
    }
#confirm:pressed {
        background-image: url(:/images/sideBar/black4.png);
    }</string>
                    </property>
                    <property name="text">
                     <string>确定</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="lineshow" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <spacer name="hsp2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="line">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>2</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>2</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#line { background:rgba(55,55,55,40); }</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hsp1">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="right" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>800</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>800</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="layout2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QWidget" name="rightcard" native="true">
              <property name="styleSheet">
               <string notr="true">#rightcard {
        background-image: url(:/images/sideBar/black2.png);
		border-radius: 12px;
    }
</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="leftMargin">
                <number>12</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>12</number>
               </property>
               <property name="bottomMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QTableView" name="tableView">
                 <property name="styleSheet">
                  <string notr="true">#tableView {                       /* ① 网格线颜色 */
    gridline-color: rgba(55,55,55,40);       /* 亮灰，可改成任意 16 进制色 / rgba() */
	border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
    font-size: 16px;

}

QHeaderView::section {

    border-image: url(:/images/sideBar/black4.png) 0 0 0 0 stretch stretch;
    color: rgb(55,55,55);
	qproperty-alignment: 'AlignCenter';   /* 水平 + 垂直居中 */
    font-size: 18px;  
}

QTableView::item {                 /* ② 普通状态文字颜色 */
    color: rgb(55,55,55);                /* 深灰 */
 	qproperty-alignment: 'AlignCenter';   /* 水平 + 垂直居中 */
    border-left: 1px solid rgba(55, 55, 55, 40); /* 给每个单元格加左边框 */

}</string>
                 </property>
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
                 </property>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="hsp4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
